### ISSUE TITLE
[Question] Possible to Dyanmically Add folders

### ISSUE THREAD

**Issue by kameshsampath** [fsnotify/fsnotify#509]
Can fsnotify `Watcher` is able to watch for folders dynamically ? I mean if my program can do `watcher.Add("dir")` when some new directories are added/created ?

**Comment by arp242** [comment]
Yes, you should always be able to add or remove paths as needed.

**Comment by kameshsampath** [comment]
thanks @arp242 . appreciate the quick response
